### ISSUE TITLE
print is used in pyhdb/protocol/parts.py

### ISSUE THREAD

**Issue by jaheba** [SAP/PyHDB#37]
print is used in ``pyhdb/protocol/parts.py`` so every execution with parameters yield an output

```
pyhdb/protocol/parts.py
337:        # print repr(payload)
349:        # print 'realobreply called with args', args
372:        # print 'realobreply unpack data called with args', len(lobdata), is_data_included, is_last_data
541:            # print 'row', num_rows, humanhexlify(payload.read())[:500]
555:        print 'writing lob buffers', row_lobs
563:            print 'Writing lob data %d bytes of %d' % (max_data_to_write, lob_buffer.encoded_lob_size)
567:                print 'adding lob_buffer to unwritte_lobs list'
```

**Comment by ralhei** [comment]
yep, sorry for that, this should be removed. Will do hopefully by the end of the week.

**Comment by ralhei** [comment]
@bsrdjan You can close this issue as those print statements have been removed (after you've accepted and merged #38)

**Comment by jarus** [comment]
Fixed by #38. Thanks @ralhei
